Hudl.Mjolnir/Command/CommandInvoker.cs

```diff
@@ -384,7 +443,7 @@ public Task<CommandResult<TResult>> InvokeReturnAsync<TResult>(AsyncCommand<TRes
 
         // failureModeForMetrics is just so we can send "throw" or "return" along with the metrics
         // event we fire for CommandInvoke(). Not really intended for use beyond that.
-        private async Task<TResult> InvokeAsync<TResult>(AsyncCommand<TResult> command, InformativeCancellationToken ct, OnFailure failureModeForMetrics)
+        private async Task<TResult> InvokeAsync<TResult>(AsyncCommand<TResult> command, OnFailure failureModeForMetrics, InformativeCancellationToken ct)
```

Comment:
These changes aren't specific to this PR, but I noticed that the `Invoke` and `InvokeAsync` methods were inconsistent with argument order.